packages/sdk-react/src/components/payables/PayableDetails/PayableDetailsInfo/PayableDetailsInfo.tsx

```diff
@@ -520,3 +558,23 @@ const PayableCounterpartName = ({
     </Stack>
   );
 };
+
+const RequiredFieldLabel = ({
```

Comment:
I would extend existing `StyledLabelTableCell`. You can just convert it to a normal component or use [`InvoiceStatusChip`](https://github.com/team-monite/monite-sdk/blob/1fd6719da4d2c73b7680afb78b1697bda098e4ca/packages/sdk-react/src/components/receivables/InvoiceStatusChip/InvoiceStatusChip.tsx#L86) as an example.